server/server.go

```diff
@@ -19,7 +19,13 @@ import (
 )
 
 // The minimum autologout duration defined in RFC 3501 section 5.4.
-const MinAutoLogout = 30 * time.Minute
+var (
+	MinAutoLogout = 30 * time.Minute
```

Comment:
Not a fan of having mutable global state like this. Maybe we can move this to the `Server` struct?

Reply:
Moved into the `Server` struct.